gateway-ha/src/test/java/io/trino/gateway/ha/router/TestTcpChecks.java

```diff
@@ -47,15 +51,15 @@ public void testConnectionCheckerSucess() throws
     ConnectionChecker checker = new ConnectionChecker();
     ConnectionCheck check = spy(checker.getChecker("abc", 1111, checkInterval, 1, 0));
     doReturn(mock(Socket.class)).when(check).makeSocket("abc", 1111);
-    Assert.assertTrue(check.tcpCheck() == ConnectionCheck.TCP_CHECK_SUCCESS);
+    assertTrue(check.tcpCheck() == ConnectionCheck.TCP_CHECK_SUCCESS);
```

Comment:
Can `AssertEquals`/`AssertNotEqual` be used here and below instead of `==`/`!=`?